City-of-Helsinki/hel-fi-drupal-grants
# fix: AU-1956, AU-1968, AU-1974: Small fixes to nuortoimpalk

## What was done


* [AU-1956](https://helsinkisolutionoffice.atlassian.net/browse/AU-1956): Page 3 tooltips
* [AU-1968](https://helsinkisolutionoffice.atlassian.net/browse/AU-1968): Page 3 and 5 character counters
* [AU-1974](https://helsinkisolutionoffice.atlassian.net/browse/AU-1974): Page 4 text update

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout feature/AU-1968-nuorisotoimipalk`
  * `make fresh`
* Run `make drush-cr`

## How to test


* [Open application](https://hel-fi-drupal-grant-applications.docker.so/fi/uusi-hakemus/nuortoimpalkka)
* Go to page 2 and fill Wage subsidy and check that you are applying for a rent grant
* Go to page 3 
* Check that "Young people’s participation times in the activities of the association" fields have tooltips
* Check that "Young people’s participation in planning and decision-making" has character counter
* Go to page 4
* Check that "Hiring costs of the previous year" has been updated according to ticket
* Go to page 5
* Check that "Additional information" has character counter
* Repeat in SV and EN


[AU-1956]: https://helsinkisolutionoffice.atlassian.net/browse/AU-1956?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[AU-1968]: https://helsinkisolutionoffice.atlassian.net/browse/AU-1968?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[AU-1974]: https://helsinkisolutionoffice.atlassian.net/browse/AU-1974?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
